inst/extdata/config_schema.json

```diff
@@ -0,0 +1,191 @@
+                "sampler_opts": {
+                    "$ref": "#/definitions/SamplerOpts"
+                }
+            },
+            "required": [
+                "as_of_date",
+                "data",
+                "disease",
+                "geo_type",
+                "geo_value",
+                "horizon",
+                "job_id",
+                "parameters",
+                "priors",
+                "sampler_opts",
+                "seed",
+                "task_id"
+            ],
+            "title": "Epinow2"
+        },
+        "Data": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "path": {
+                    "type": "string"
+                },
+                "blob_storage_container": {
+                    "type": ["null", "string"]
+                },
+                "report_date": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "date"
+                    }
+                },
+                "reference_date": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "date"
+                    }
+                }
+            },
+            "required": [
+                "blob_storage_container",
+                "path",
+                "reference_date",
+                "report_date"
+            ],
+            "title": "Data"
+        },
+        "Parameters": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "path": {
+                    "type": "string"
+                },
+                "blob_storage_container": {
+                    "type": ["string", "null"]
+                }
+            },
+            "required": [
+                "blob_storage_container",
+                "path"
+            ],
+            "title": "Parameters"
+        },
+        "Priors": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "rt": {
+                    "$ref": "#/definitions/Rt"
+                },
+                "gp": {
+                    "$ref": "#/definitions/Gp"
+                }
+            },
+            "required": [
+                "gp",
+                "rt"
+            ],
+            "title": "Priors"
+        },
+        "Gp": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "alpha_sd": {
+                    "type": "number"
+                }
+            },
+            "required": [
+                "alpha_sd"
+            ],
+            "title": "Gp"
+        },
+        "Rt": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
```

Comment:
Can we add properties other than mean and sd? Is that something we'd ever do? Or would that be a big enough change that it would require changing the schema anyway?

Reply:
No, [EpiNow2::rt_opts()](https://epiforecasts.io/EpiNow2/reference/rt_opts.html) only takes mean and sd as priors.